packages/app/src/sandbox/eval/presets/preact-cli/preset.js

```diff
@@ -0,0 +1,118 @@
+import babelTranspiler from '../../transpilers/babel';
+import jsonTranspiler from '../../transpilers/json';
+import stylesTranspiler from '../../transpilers/style';
+import sassTranspiler from '../../transpilers/sass';
+import rawTranspiler from '../../transpilers/raw';
+import stylusTranspiler from '../../transpilers/stylus';
+import lessTranspiler from '../../transpilers/less';
+
+import asyncTranspiler from './transpilers/async';
+
+import Preset from '..';
+
+export default function PreactPreset() {
+  const preactPreset = new Preset(
+    'preact-cli',
+    ['js', 'jsx', 'ts', 'tsx', 'json', 'less', 'scss', 'sass', 'styl', 'css'],
+    {
+      preact$: 'preact',
+      // preact-compat aliases for supporting React dependencies:
+      react: 'preact/compat',
+      'react-dom': 'preact/compat',
+      'create-react-class': 'preact/compat/lib/create-react-class',
+      'react-addons-css-transition-group': 'preact-css-transition-group',
+    }
+  );
+
+  preactPreset.registerTranspiler(module => /\.m?jsx?$/.test(module.path), [
+    {
+      transpiler: babelTranspiler,
+      options: {
+        isV7: false,
```

Comment:
I believe that the newest `preact-cli` is on version 7 of Babel nowadays, would be good to check their babel config and update this one to the new one.

Reply:
updated but now I get a babel error

Reply:
pr4914.build.csb.dev/s/preact-template-compat-issue-forked-jd3yo?file=/src/index.js